waku/waku_rln_relay/constants.nim

```diff
@@ -37,18 +37,18 @@ const
 
 # temporary variables to test waku-rln-relay performance in the static group mode
 const
-  StaticGroupSize* = 10000
-  #  StaticGroupKeys is a static list of 10000 membership keys in the form of (identity trapdoor, identity nullifier, identity secret hash, identity commitment)
+  OffchainGroupSize* = 10000
```

Comment:
`Static` was a better name. But this is test data, so I would name it like that.